Description: Using python -m coverage instead of /usr/bin/coverage
Author: Steven Hardy <shardy@redhat.com>
Origin: upstream, https://review.openstack.org/#/c/52668/
Bug-Ubuntu: https://launchpad.net/bugs/1241330
Last-Update: 2013-10-20

--- heat-2013.2.orig/run_tests.sh
+++ heat-2013.2/run_tests.sh
@@ -117,7 +117,7 @@ fi
 if [ "$coverage" == 1 ]; then
     echo "Generating coverage report in ./cover"
     python setup.py testr --coverage --slowest
-    coverage report -m
+    python -m coverage report --show-missing
 fi
 
 exit $result
